pkg/settings/setting.go

```diff
@@ -118,7 +118,7 @@ var (
 	PartnerChartDefaultBranch           = NewSetting("partner-chart-default-branch", "main")
 	RKE2ChartDefaultBranch              = NewSetting("rke2-chart-default-branch", "main")
 	FleetDefaultWorkspaceName           = NewSetting("fleet-default-workspace-name", fleetconst.ClustersDefaultNamespace) // fleetWorkspaceName to assign to clusters with none
-	ShellImage                          = NewSetting("shell-image", "rancher/shell:v0.1.20")
+	ShellImage                          = NewSetting("shell-image", "rancher/shell:v0.1.21-rc1")
```

Comment:
Why are we doing this ?

Reply:
I have no idea, but I will force it back